contracts/meta-oracles/HistoricalPriceFeedV2.sol

```diff
@@ -55,24 +64,24 @@ contract HistoricalPriceFeedV2 is
      * Stores Historical prices according to passed in oracle address. Updates must be 
      * triggered off chain to be stored in this smart contract. This contract negates the problem
      * of drift by allowing price feed updates on a predetermined cadence based on the time of deployment.
-     * This means delays in calling poke do not propogate throughout the whole dataset and the drift caused
+     * This means delays in calling poke do not propagate throughout the whole dataset and the drift caused
      * by previous poke transactions not being mined exactly on nextAvailableUpdate do not compound
-     * as they would if it was required that poke is called an updateFrequency amount of time after
+     * as they would if it was required that poke is called an updatePeriod amount of time after
      * the last poke.
      *
-     * @param  _updateFrequency           Cadence at which data is allowed to be logged, based off 
-                                          deployment timestamp 
+     * @param  _updatePeriod              Cadence (in seconds) at which data is allowed to be logged,
```

Comment:
This is partly incorrect, because data can be logged more frequently than that. Currently `updatePeriod` is the amount of time such that at any moment since deployment the average gap between actual poke moments is larger than that.